core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -436,7 +436,8 @@ public void close(ScheduledExecutorService instance) {
    */
   @VisibleForTesting
   static class TimeoutMarshaller implements Metadata.AsciiMarshaller<Long> {
-    private static final BiMap<TimeUnit, Character> UNITS =
+    // Bimaps have consistent iteration order.
```

Comment:
s/Bimaps/ImmutableBiMaps/

The Bimap iterface does not define a consistent iteration order.